src/index/IndexImpl.Text.cpp

```diff
@@ -1484,7 +1466,7 @@ void IndexImpl::getECListForWordsAndSingleSub(
 
   LOG(DEBUG) << "Filtering matching contexts and building cross-product...\n";
   vector<array<Id, 3 + I>> nonAggRes;
-  if (cids.size() > 0) {
+  if (wep.cids.size() > 0) {
```

Comment:
I am sure, static analysis also tells you this.
```suggestion
  if (!wep.cids.empty() {
```